cbmc/proofs/IotMqtt_Wait/Makefile

```diff
@@ -38,6 +38,9 @@ DEF += -DTOPIC_LENGTH_MAX=$(TOPIC_LENGTH_MAX)
 LOOP += valid_IotMqttSubscriptionList.0:$(SUBSCRIPTION_COUNT_MAX)
 LOOP += valid_IotMqttOperationList.0:$(SUBSCRIPTION_COUNT_MAX)
 LOOP += IotListDouble_RemoveAllMatches.0:$(SUBSCRIPTION_COUNT_MAX)
+LOOP += IotListDouble_FindFirstMatch.0:$(SUBSCRIPTION_COUNT_MAX)
```

Comment:
What is the problem that adding these loop limits solves?

Reply:
This is renaming of static loops in the new version of cbmc.